deploy/deploy.go

```diff
@@ -56,6 +56,14 @@ type Deployer struct {
 	force         bool             // true forces upload of all files
 	invalidateCDN bool             // true enables invalidate CDN cache (if possible)
 	maxDeletes    int              // caps the # of files to delete; -1 to disable
+
+	// For tests...
+	summary    deploySummary // summary of latest Deploy results
+	testBucket *blob.Bucket  // injected bucket
```

Comment:
I'm not a fan of having different code paths for test. I suggest that you add a `openBucket` field (func) to the `Deployer` struct.

Reply:
Done.